src/calculateBullAndCows.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const calculateBullAndCows = (correct, guess) => [...guess].reduce(
```

Comment:
```suggestion
const calculateBullAndCows = (correct, guess) => guess.reduce(
```
`reduce()` is non-mutable method